lib/index.js

```diff
@@ -127,3 +128,107 @@ $('#foodFilter').on('keyup', filterFoods);
 
 
 module.exports = {addNewFood}
+
+
+
+
+
+// Meals
+
+const getMeals = () => {
+  return fetch(`https://evening-peak-25213.herokuapp.com/api/v1/meals`)
+    .then(response => handleResponses(response))
```

Comment:
I wonder if we can later pull this out to a module or something like that so that the code can get both.